.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: go
+
+go:
+  - "1.11.x"
+  - master
+
+env:
+  - DEP_VERSION=0.5.0
+  - NODE_TEST_IMAGE_VERSION=10.15.0
```

Comment:
Do we need this env var? and DEV_VERSION?

Reply:
Those are versions which may change with the course of time as `dep` hasn't released even the first version, and the other is just a test image which will be excluded from travis file once the issue #11 has been solved. I prefer to keep the versions as environment variables.